src/react/package.json

```diff
@@ -3,19 +3,25 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "eslint-config-react": "^1.1.7",
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
     "react-router-dom": "^4.3.1",
     "react-scripts": "2.1.3"
   },
+  "devDependencies": {
```

Comment:
we dont need to add eslint / airbnb / plugins here, since the config for the base project can be extended

Reply:
Thanks! Removed!